setup.py

```diff
@@ -56,11 +56,14 @@
             "sphinx_paramlinks",
             "changelog",
             "flake8",
-            "black",
+            "black~=24.0",
             "rope",
             "doc8",
             "pre-commit",
-        ]
+        ],
+        "aws": [
+            "dask-cloudprovider[aws]",
```

Comment:
Maybe add this to "dev" too so that "dev" includes everything?

Reply:
So we don't get repetitive there, I'd rather not. You can install both with `pip install ".[dev,aws]"`